From 4961cde862ea162f4a4318a252681ef23cf10538 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Mon, 14 May 2018 11:05:15 +0800
Subject: [PATCH 3783/5242] MLK-18300-1 ARM64: dts: freescale: imx8qm: remove
 csi gpio clocks

commit  affefe61a6c6bfe8eea4e8c8b1d5c5773c7c3415 from
https://source.codeaurora.org/external/imx/linux-imx.git

Remove CSI0/1 GPIO related clocks to make sure all GPIOs
clocks are always ON.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-device.dtsi      |   12 ++++--------
 1 file changed, 4 insertions(+), 8 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
index 3967e4d..2d7e862 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
@@ -2452,7 +2452,7 @@
 				<0x0 0x58221000 0x0 0x1000>; /* CSI0 Subsystem CSR base addr  */
 			interrupts = <10 IRQ_TYPE_LEVEL_HIGH>;
 			interrupt-parent = <&irqsteer_csi0>;
-			clocks = <&clk IMX8QM_CSI0_APB_CLK>,
+			clocks = <&clk IMX8QM_CLK_DUMMY>,
 					<&clk IMX8QM_CSI0_CORE_CLK>,
 					<&clk IMX8QM_CSI0_ESC_CLK>,
 					<&clk IMX8QM_IMG_PXL_LINK_CSI0_CLK>;
@@ -2470,7 +2470,7 @@
 					<0x0 0x58241000 0x0 0x1000>; /* CSI1 Subsystem CSR base addr  */
 			interrupts = <10 IRQ_TYPE_LEVEL_HIGH>;
 			interrupt-parent = <&irqsteer_csi1>;
-			clocks = <&clk IMX8QM_CSI1_APB_CLK>,
+			clocks = <&clk IMX8QM_CLK_DUMMY>,
 					<&clk IMX8QM_CSI1_CORE_CLK>,
 					<&clk IMX8QM_CSI1_ESC_CLK>,
 					<&clk IMX8QM_IMG_PXL_LINK_CSI1_CLK>;
@@ -2695,7 +2695,7 @@
 		interrupt-controller;
 		interrupt-parent = <&gic>;
 		#interrupt-cells = <2>;
-		clocks = <&clk IMX8QM_CSI0_LIS_IPG_CLK>;
+		clocks = <&clk IMX8QM_CLK_DUMMY>;
 		clock-names = "ipg";
 		power-domains = <&pd_csi0>;
 	};
@@ -2721,7 +2721,7 @@
 		interrupt-controller;
 		interrupt-parent = <&gic>;
 		#interrupt-cells = <2>;
-		clocks = <&clk IMX8QM_CSI1_LIS_IPG_CLK>;
+		clocks = <&clk IMX8QM_CLK_DUMMY>;
 		clock-names = "ipg";
 		power-domains = <&pd_csi1>;
 	};
@@ -3068,8 +3068,6 @@
 		interrupt-controller;
 		#interrupt-cells = <2>;
 		power-domains = <&pd_csi0>;
-		clocks = <&clk IMX8QM_CSI0_IPG_CLK_S>;
-		clock-names = "ipg";
 		status = "disabled";
 	};
 
@@ -3083,8 +3081,6 @@
 		interrupt-controller;
 		#interrupt-cells = <2>;
 		power-domains = <&pd_csi1>;
-		clocks = <&clk IMX8QM_CSI1_IPG_CLK_S>;
-		clock-names = "ipg";
 		status = "disabled";
 	};
 
-- 
1.7.9.5

